src/dashboard/home-dashboard.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import HomeDashboard from "./home-dashboard.component";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+const match = { params: { id: 1 }, isExact: true, path: "", url: "" };
+function renderWithRouter(
```

Comment:
this function looks like it was painful to come up with. Tests can be tricky sometimes 😢